<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Catalog</class>
 <widget class="QDialog" name="Catalog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>1019</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1240</width>
    <height>313</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1240</width>
    <height>20000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Выйти</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Арендовать авто</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="minimumSize">
      <size>
       <width>1220</width>
       <height>267</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>2000</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-1349</y>
        <width>1230</width>
        <height>2363</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0,0,0,0" columnstretch="0,0,0,0">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="horizontalSpacing">
        <number>-1</number>
       </property>
       <item row="10" column="0">
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>1200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:700; color:#ffd479;&quot;&gt;RentAuto&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; color:#babdbe;&quot;&gt;Аренда премиальных автомобилей&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="indent">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="bmw_xm">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>267</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>267</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>pictures/bmw_xm-2.jpg</pixmap>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="mercedes_maybach">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>267</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>267</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>pictures/mercedes_maybach-2.jpg</pixmap>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="Lamborghini_huracan">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>267</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>267</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>pictures/lamborghini_huracan-2.jpg</pixmap>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="audi_a5">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>267</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>267</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>pictures/audi_a5-2.jpg</pixmap>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="dodge_challenger">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>267</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>267</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>pictures/Dodge_challenger-2.jpg</pixmap>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="bmw_m5">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>267</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>267</height>
          </size>
         </property>
         <property name="font">
          <font>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>pictures/bmw_m5_competition_2.jpg</pixmap>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="lamborghini_urus">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>267</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>267</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>pictures/lamborghini_urus-2.jpg</pixmap>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="4">
        <widget class="QLabel" name="information">
         <property name="maximumSize">
          <size>
           <width>1200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:700;&quot;&gt;Прокат авто без водителя в Петрозаводске&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Компания &amp;quot;RentAuto&amp;quot; оказывает услуги посуточной и долгосрочной аренды автомобилей в Петрозаводске.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Наш автопрокат предлагает легковые машины на любой бюджет, от недорогого седана комфорт-класса до элитного суперкара.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Стоимось проката авто зависит от срока использования - чем больше дней, тем дешевле будет стоимость суток аренды.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:700;&quot;&gt;Оформление аренды&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Чтобы взять у нас автомобиль в аренду, Вам понадобиться только паспорт РФ и водительские права.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Оформление займет всего 15 минут, после чего Вы сможете получить автомобиль в любом удобном Вам районе города.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Вам достаточно оставить заявку в приложении или связаться с нашими манеджерами по телефону, все остальное сделаем мы.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Договор аренды подписывается при получение машины.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:700;&quot;&gt;Дополнительные услуги&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;&amp;quot;RentAuto&amp;quot; предлагает дополнительные услуги, чтобы аренда автомобиля в нашей компании была для Вас максимально комфортной и оставила только приятные воспоминания:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;-Полное КАСКО. Весь автопарк компании полностью застрахован.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;-Техническая поддержка 24/7. Мы всегда рядом, если Вам потребуется помощь или консультация.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;-Прокат автомобиля с водителем. Если Вам требуется персональный водитель, компания может предоставить его.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;-Доставка по адресу. После оформления аренды мы доставим автомобиль по указанному адресу в Петрозаводске.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;-Мы гарантируем вашу конфиденциальность. Ни один прокатный автомобиль не оборудован средствами аудио-видеофиксации.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <property name="indent">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; color:#ffffff;&quot;&gt;Контакты&lt;/span&gt;&lt;/p&gt;&lt;p&gt;🏢 &lt;span style=&quot; color:#b5b8b9;&quot;&gt;Проспект Ленина., д. 9&lt;/span&gt;&lt;/p&gt;&lt;p&gt;🕓 &lt;span style=&quot; color:#babdbe;&quot;&gt;Без выходных, Круглосуточно&lt;/span&gt;&lt;/p&gt;&lt;p&gt;📱 &lt;span style=&quot; color:#babdbe;&quot;&gt;+7(777) 777-77-77&lt;/span&gt;&lt;/p&gt;&lt;p&gt;📩 &lt;span style=&quot; color:#babdbe;&quot;&gt;info@rentauto.ru&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="Mercedes_g63">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>267</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>267</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>pictures/Mercedes_g63_amg_2.jpg</pixmap>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="bmw_z4">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>267</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>267</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>pictures/bmw_z4-2.jpg</pixmap>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#babdbe;&quot;&gt;С 2024. RentAuto.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLabel" name="porsche_718">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>267</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>267</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>pictures/Porsche_718-2.jpg</pixmap>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="Rolls_Royce_Wraith">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>267</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>267</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>pictures/Rolls_Royce_Wraith-2.jpg</pixmap>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="Zeekr_001">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>267</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>267</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>pictures/Zeekr_001_2.jpg</pixmap>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="bmw_m3">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>267</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>267</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>pictures/bmw_m3-2.jpg</pixmap>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="Bentley_Continental">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>267</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>267</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>pictures/Bentley_Continental-2.jpg</pixmap>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="bmw_m8">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>267</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>267</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>pictures/bmw_m8-2.jpg</pixmap>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QLabel" name="Chevrolet_camaro">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>267</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>267</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>pictures/camaro-2.jpg</pixmap>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QLabel" name="lixiang_L9">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>267</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>267</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>pictures/lixiang_l9-2.jpg</pixmap>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QLabel" name="porsche_macan">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>267</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>10000</width>
           <height>10000</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>pictures/Porsche_macan-2.jpg</pixmap>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
